htlcswitch/circuit_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/lightningnetwork/lightning-onion"
+	"github.com/jimpo/lightning-onion"
```

Comment:
Note that for future ref, updating the glide file to modify the _source_ is enough.